description: Link against libharfbuzz-subset when use_system_harfbuzz

--- a/third_party/harfbuzz-ng/BUILD.gn
+++ b/third_party/harfbuzz-ng/BUILD.gn
@@ -16,7 +16,10 @@
       "//third_party:freetype_harfbuzz",
       "//third_party/freetype:freetype_source",
     ]
-    packages = [ "harfbuzz" ]
+    packages = [
+      "harfbuzz",
+      "harfbuzz-subset",
+    ]
   }
 } else {
   config("harfbuzz_config") {
