description: disable building font tests
author: Michael Gilbert <mgilbert@debian.org>

--- a/skia/BUILD.gn
+++ b/skia/BUILD.gn
@@ -850,7 +850,7 @@ group("test_fonts_resources") {
   if (is_mac || is_ios) {
     deps += [ ":test_fonts_bundle_data" ]
     data_deps += [ ":test_fonts_bundle_data" ]
-  } else {
+  } else if (false) {
     deps += [ "//third_party/test_fonts" ]
     data_deps += [ "//third_party/test_fonts" ]
   }
@@ -890,7 +890,7 @@ source_set("test_fonts") {
       "//third_party/fuchsia-sdk/sdk:sys_cpp",
     ]
   }
-  if (is_linux) {
+  if (false) {
     sources += [ "ext/test_fonts_linux.cc" ]
   }
   if (is_mac) {
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -242,7 +242,7 @@ static_library("test_support") {
     deps += [ "//third_party/fuchsia-sdk/sdk:zx" ]
   }
 
-  if (is_linux) {
+  if (false) {
     public_deps += [ ":fontconfig_util_linux" ]
     data_deps = [ "//third_party/test_fonts" ]
     if (current_toolchain == host_toolchain) {
@@ -411,7 +411,7 @@ if (is_linux) {
     outputs = [ "${root_build_dir}/etc/fonts/{{source_file_part}}" ]
   }
 
-  if (current_toolchain == host_toolchain) {
+  if (false) {
     executable("generate_fontconfig_caches") {
       testonly = true
       sources = [ "generate_fontconfig_caches.cc" ]
--- a/content/shell/app/shell_main_delegate.cc
+++ b/content/shell/app/shell_main_delegate.cc
@@ -293,7 +293,7 @@ bool ShellMainDelegate::BasicStartupComp
     BlinkTestPlatformInitialize();
 #endif
 
-#if !defined(OS_WIN)
+#if 0
     // On Windows BlinkTestPlatformInitialize() is responsible for font
     // initialization.
     skia::ConfigureTestFont();
